gateway/src/main/java/org/georchestra/gateway/security/GatewaySecurityConfiguration.java

```diff
@@ -103,6 +117,12 @@ private Stream<ServerHttpSecurityCustomizer> sortedCustomizers(List<ServerHttpSe
         return new ResolveGeorchestraUserGlobalFilter(resolver);
     }
 
+    @ConditionalOnExpression("${georchestra.gateway.headerAuthentication:false}")
```

Comment:
It would make more sense to have the property onto `georchestra.gateway.security.headerAuthentication` in my opinion